src/MTGCore/Controllers/CardController.cs

```diff
@@ -53,16 +58,33 @@ public async Task<ActionResult> Details(int id)
         {
             var response = await _mtgService.GetCardByID(id);
 
-            //string rootPath = _env.WebRootPath;
 
             response.manaCost = _conversion.ConvertToSymbol(response.manaCost);
 
-            var model = _mapper.Map<Cards>(response);
+            var model = _mapper.Map<CardDto>(response);
+
+            var UserIDString = _userManager.GetUserId(HttpContext.User);
+
+            var userId = new Guid(UserIDString);
+
```

Comment:
Is storing IDs as GUIDs better than storing them as integers?

Reply:
no its just that microsofts identity is set up to use GUIDS for some reason.

https://dba.stackexchange.com/a/266

good post on pros and cons. May Refactor in the future if its easy to do with identity